app/create_db_tables.py

```diff
@@ -10,37 +10,38 @@
 
 def initialize_police_table():
     load_dotenv()
-    police_table = """CREATE TABLE IF NOT EXISTS police_force (
-    id SERIAL PRIMARY KEY NOT NULL,
-    dates TIMESTAMP,
-    links TEXT,
-    case_id TEXT,
-    city TEXT,
-    state TEXT,
-    title TEXT,
-    description TEXT,
-    tags TEXT,
-    force_rank TEXT,
-    confidence TEXT
-        );"""
+    # police_table = """CREATE TABLE IF NOT EXISTS police_force (
+    # id SERIAL PRIMARY KEY NOT NULL,
+    # dates TIMESTAMP,
+    # links TEXT,
+    # case_id TEXT,
+    # city TEXT,
+    # state TEXT,
+    # title TEXT,
+    # description TEXT,
+    # tags TEXT,
+    # force_rank TEXT,
+    # confidence TEXT
+    #     );"""
 
-    pi_table = """CREATE TABLE IF NOT EXISTS incidents (
-    id SERIAL PRIMARY KEY NOT NULL,
-    tweet_id TEXT,
+    pi_table = """CREATE TABLE IF NOT EXISTS twitter_incidents (
+    incident_id SERIAL PRIMARY KEY NOT NULL,
     date_created TIMESTAMP,
+    tweet_id TEXT,
     user_name TEXT,
-    user_description TEXT,
-    twitter_text TEXT,
-    force_rank TEXT,
-    confidence TEXT,
-    tags TEXT,
+    description VARCHAR(10000),
     city TEXT,
     state TEXT,
-    twitterbot_tweet_id TEXT,
-    responses TEXT
+    lat FLOAT,
+    long FLOAT,
+    title TEXT,
+    force_rank TEXT,
+    status TEXT,
+    confidence FLOAT,
+    tags TEXT
           );"""
     
-    db_url = os.getenv('DB_URL')
+    db_url = os.getenv('HER_URL')
```

Comment:
In the scraper.py we have this set to the "DB_URI" credential - should they all be the same?

Reply:
will change to DB_URI